openqa-maintenance.py

```diff
@@ -505,9 +505,10 @@ def calculate_incidents(self, incidents):
         """
         l_incidents=[]
         for kind, prj in incidents.items():
-            # TODO: unfortuanetly this works correctly only for IBS ( OBS using
-            # incident_NR )
-            l_incidents.append((kind + '_TEST_ISSUES', ','.join([x.split('.')[1] for x in osc.core.meta_get_packagelist(self.apiurl, prj)])))
+            l_incidents.append(
+                ( kind + '_TEST_ISSUES', ','.join(
+                    [x.replace('_', '.').split('.')[1] for x in osc.core.meta_get_packagelist(self.apiurl, prj)]
+                )))
```

Comment:
this wasn't exactly the meaning of split I was refering too - some local variables to make this more readable. But next round...